tests/chart_tests/test_elasticsearch.py

```diff
@@ -704,3 +706,20 @@ def test_elasticsearch_statefulset_with_scc_enabled(self, kube_version):
         assert docs[0]["apiVersion"] == "security.openshift.io/v1"
         assert docs[0]["metadata"]["name"] == "release-name-elasticsearch-anyuid"
         assert docs[0]["users"] == ["system:serviceaccount:default:release-name-elasticsearch"]
+
+    def test_elasticsearch_without_controlplane_flag_disabled(self, kube_version):
+        """Test that helm renders the no templates when controlplane is disabled."""
+        doc_dir = Path(f"{git_root_dir}/charts/elasticsearch/templates")
+        all_docs = []
+
+        for subdir in doc_dir.iterdir():
+            if subdir.is_dir():
+                yaml_files = [str(x.relative_to(git_root_dir)) for x in subdir.glob("*.yaml")]
+                all_docs.extend(yaml_files)
```

Comment:
Comprehensions are tidier and faster than create-then-modify <https://youtu.be/YY7yJHo0M5I?t=829>

```suggestion
        all_docs = [
            str(path.relative_to(git_root_dir))
            for path in list(
                git_root_dir.rglob("charts/elasticsearch/templates/*/*.yaml")
            )
        ]
```